pkg/diff/chunk.go

```diff
@@ -332,36 +340,37 @@ func (s *randomSpliter) splitRange(db *sql.DB, chunk *chunkRange, count int, sch
 
 			// already have the chunk [column = value], so next chunk should start with column > value
 			lowerSymbol = gt
-		} else {
-			if i == 0 {
-				if useNewColumn {
-					lower = ""
-					lowerSymbol = ""
-				} else {
-					lower = splitValues[i]
-					lowerSymbol = symbolMin
-				}
+		}
+
+		if i == 0 && valueCounts[i] == 1 {
```

Comment:
it's very unfriendly to reviewer, please refine these code, make the logic more clear